lib/ability.rb

```diff
@@ -15,10 +15,18 @@ def initialize(user, resource, controller)
     return unless user
 
     if user.administrator?
-      can :manage, :all
+      if resource.is_a?(PriceGroup)
+        can :manage_members, resource if resource.admin_editable?
+      else
+        can :manage, :all
+      end
       return
     end
 
+    if resource.is_a?(PriceGroup) && !resource.global? && user.manager_of?(resource.facility)
```

Comment:
I considered changing `#manager_of?` to return `false` if `facility` is `nil` but that wouldn't be right, since administrators are managers of global price groups... whatever I've thought of so far is worse than this line, in other words.